Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -62,11 +62,7 @@ public final class ProjectPamphletViewController: UIViewController {
   }
 
   private var initialTopConstraint: CGFloat {
-    if #available(iOS 11.0, *) {
-      return parent?.view.safeAreaInsets.top ?? 0.0
-    } else {
-      return UIApplication.shared.statusBarFrame.size.height
-    }
+    return self.parent?.view.safeAreaInsets.top ?? 0.0
```

Comment:
I think here we should just be able to use `self.view.safeAreaInsets.top` as well?